docs/extensions/ListViewExtensions.md

```diff
@@ -1,101 +1,217 @@
+```
+
+### Sample Output
+
+![SmoothScrollIntoView Helper](../resources/images/Extensions/SmoothScrollIntoView.gif)
+
+### Methods
+
+| Methods | Return Type | Description |
+| -- | -- | -- |
+| SmoothScrollIntoViewWithIndexAsync(int, ScrollItemPlacement, bool, bool, int, int) | Task | Smooth scroll item into view With index number |
+| SmoothScrollIntoViewWithItemAsync(object, ScrollItemPlacement, bool, bool, int, int) | Task | Smooth scroll item into view With item object |
+
+### Method params
+
+| Properties | Type | Description |
+|------------|------|-------------|
+| intex      | int  | Intex of the item to be scrolled. Index can be negative |
+| item      | int  | Intex of the item to be scrolled |
+| itemPosition | ScrollItemPlacement | Specify the position of the Item after scrolling |
+| disableAnimation | bool | To disable the scrolling animation |
+| scrollIfVisibile | bool | Set `true` to scroll even if the scroll to item is visible so that the item will be aligned depend upon `itemPosition` |
+| additionalHorizontalOffset | bool | Give addition horizontal offset |
+| additionalVerticalOffset | bool | Give addition vertical offset |
+
+### ItemPosition
+
+| ScrollItemPlacement | Description |
+|--------------|-------------|
+| Default | If visible then it will not scroll, if not then item will be aligned to the nearest edge |
+| Left | Aligned left |
+| Top | Aligned top |
+| Centre | Aligned centre |
+| Right | Aligned right |
+| Bottom | Aligned bottom |
+
+### Examples
+
+- We can use this extension to make the selected item always centered.
+
+    **Sample Code**
+
+    ```xaml
+    <ListView ItemsSource="{x:Bind itemSources}" SelectionChanged="ListView_SelectionChanged">
+        <ListView.ItemTemplate>
+            <DataTemplate>
+                <!-- Your Template -->
+            </DataTemplate>
+        </ListView.ItemTemplate>
+        <ListView.ItemsPanel>
+            <ItemsPanelTemplate>
+                <StackPanel Orientation="Horizontal"/>
+            </ItemsPanelTemplate>
+        </ListView.ItemsPanel>
+    </ListView>
+    ```
+
+    ```csharp
+    private async void ListView_SelectionChanged(object sender, SelectionChangedEventArgs e)
+    {
+        var listView = (sender as ListView);
+        await listView.SmoothScrollIntoViewWithIndex(listView.SelectedIndex, ItemPlacement.Centre, false, true);
+    }
+    ```
+
+    ```vb
+    Private Async Sub ListView_SelectionChanged(ByVal sender As Object, ByVal e As SelectionChangedEventArgs)
+        Dim listView = (TryCast(sender, ListView))
+        Await listView.SmoothScrollIntoViewWithIndex(listView.SelectedIndex, ItemPlacement.Centre, False, True)
+    End Sub
+    ```
+
+    **Sample Output**
+
+    ![Use Case 1 Output](../resources/images/Extensions/SmoothScrollIntoView-CenterSelected.gif)

```

Comment:
Not sure if we can use these icons even in an example, plus the file is a bit large still. Do we think we need an example image here if we have the other one?

Reply:
@Vijay-Nirmal not sure if you missed this, mind rendering a smaller file sized version with non-descript images (basic shapes or something)?

Reply:
@michael-hawker I forgot about it, I will do it ASAP

Reply:
I think being closer to the half-meg that your other image is would be acceptable if it's animated still?

Reply:
I have uploaded a new Gif with no logo and the file size is 600KB. Hope its fine